example/code/code.go

```diff
@@ -0,0 +1,9 @@
+package code
+
+// Return codes for the examples
+const (
+	CodeTypeOK            uint32 = 0
```

Comment:
you can use `iota` https://dlintw.github.io/gobyexample/public/constants-and-iota.html

Reply:
I don't like iota for things that are cross-language standards because then you have to count to figure out which one is which. Granted it's ok here since there's so few and it's easier to maintain.

Reply:
True true